tests/test_omen.py

```diff
@@ -190,6 +191,39 @@ def test_bigtx_commit():
     assert car2.gas_level == 9
 
 
+@patch("omen2.object.VERY_LARGE_LOCK_TIMEOUT", 3)
+def test_tabtx_no_deadlock():
```

Comment:
we probably need a test that proves that obj1 can be modified while obj2 is locked.

Reply:
added `test_per_obj_locks()`